docs-src/cloud/how-to-setup-export.md

```diff
@@ -0,0 +1,111 @@
+---
+id: how-to-configure-export
+title: How to configure Workflow History Export
+sidebar_label: Configure Workflow History Export
```

Comment:
These result in anchors -
The anchors themselves can be short and static - 
We can change titles without fear of creating a bad link.
```suggestion
sidebar_label: Configure
```